index.js

```diff
@@ -147,4 +164,12 @@ export default class Channel {
       }
     }
   }
+
+  /**
+   * Dispatch accumulated events
+   * @private
+   */
+  _dispatchAccumulated () {
+    this._accumulatedEvents.forEach(args => this.dispatch.apply(this, args));
```

Comment:
i suggest to clear `this._accumulatedEvents` here instead of in `unmute ()` as it seems to me more consistent to always clear list after dispatching.
```js
this._accumulatedEvents.lenght = 0;
```